models/articles.model.js

```diff
@@ -10,12 +10,21 @@ exports.selectArticleById = (article_id) => {
       return rows[0];
     });
 };
-exports.selectArticles = () => {
+exports.selectArticles = (topic) => {
+    const validTopics = ['cats', 'mitch', 'test', 'coding', 'football', 'cooking']
```

Comment:
What if we add an endpoint in the future that posts new topics this is not a dynamic way of checking for a valid topic. You could query the DB though to see if the topic exists.